Frontend/CO.CDP.OrganisationApp/Pages/Supplier/ConnectedEntity/ConnectedEntityCheckAnswersOrganisation.cshtml.cs

```diff
@@ -154,6 +192,139 @@ private void InitModal(ConnectedEntityState state)
         return registerConnectedEntity;
     }
 
+    private UpdateConnectedEntity? UpdateConnectedEntityPayload(ConnectedEntityState state)
+    {
+        UpdateConnectedOrganisation? connectedOrganisation = null;
+        UpdateConnectedIndividualTrust? connectedIndividualTrust = null;
+
+        if (state.ConnectedEntityType == Constants.ConnectedEntityType.Organisation)
+        {
+            connectedOrganisation = new UpdateConnectedOrganisation
+            (
+                category: state.ConnectedEntityOrganisationCategoryType!.Value.AsApiClientConnectedEntityOrganisationCategoryType(),
+                controlCondition: state.ControlConditions.AsApiClientControlConditionList(),
+                insolvencyDate: state.InsolvencyDate,
+                lawRegistered: state.LawRegistered,
+                name: state.OrganisationName,
+                organisationId: null,
+                registeredLegalForm: state.LegalForm
+            );
+        }
+        else
```

Comment:
we can remove this block from here as this page only for Organisation check answer